src/main/python/sunburst_visualization/sunburst.py

```diff
@@ -0,0 +1,98 @@
+import plotly.express as px
+import sys
+import json
+import os
+
+class SunBurstVisualization:
+    def __init__(self, project_path):
+
+        f = open(project_path + os.path.sep + "results.json")
+
+        self.result = json.load(f)
+
+        def createsunburst(self, result):
+            self.result=data
+
+        character_ = []
+        parent_ = []
+        values_ = []
+        act_parent = None
+
+        for files in self.result["files"]:
+
+            for k,v in files.items():
+                if k == "relativePath":
+                    character_.append(v)
+                    parent_.append("")
+                    values_.append(0)
+                    act_parent = v
+
+                if isinstance(v, list):
+                    classes = v
+                    for listaelem_class in classes:
+                        if listaelem_class["name"] == "":
+                            for metodus_elem in listaelem_class["methods"]:
+                                if metodus_elem["name"] == "":
+                                    character_.append(files["relativePath"]+("\\")+listaelem_class["name"]+("\\")+("None_method"))
```

Comment:
I don't think None_method is the right word that should go here. 
Try something like ""